### PR TITLE
Cleaned C&C yaml rules

### PR DESCRIPTION
**Pull request by reaperrr**
Streamlines defaults and removes some duplication.

### CODE REVIEW

**Comment by reaperrr** [comment]
Rebased.

**Comment by Mailaender** [comment]
Can't spot any regressions. :white_check_mark:

File: mods/cnc/rules/defaults.yaml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+^ExistsInWorld:
+	AppearsOnRadar:
+	UpdatesPlayerStatistics:
+	CombatDebugOverlay:
+	DrawLineToTarget:
```
Comment (GraionDilach) [inline mods/cnc/rules/defaults.yaml]: This was removed from here in the TS cleanup. Let's aim for consistency.

File: mods/cnc/rules/defaults.yaml [hunk 2]
```diff
@@ -593,43 +580,49 @@
...
 	TransformOnCapture:
 		ForceHealthPercentage: 25
-	BodyOrientation:
 	Tooltip:
 		GenericName: Destroyed Vehicle
 	DisabledOverlay:
 	ScriptTriggers:
+	WithFacingSpriteBody:
+	AutoSelectionSize:
+	RenderSprites:
```
Comment (GraionDilach) [inline mods/cnc/rules/defaults.yaml]: ^CommonHuskDefaults is shared between ^Husk and ^HelicopterHusk so why aren't these three lines shared as well?

**Comment by GraionDilach** [comment]
Also you removed UnitExplodeSmall from husks which might be undesirable due to it was being used as a visual effect mostly.

File: mods/cnc/rules/defaults.yaml (hunk 2 above)
Comment (reaperrr) [inline mods/cnc/rules/defaults.yaml]: Good question, will fix that.

**Comment by reaperrr** [comment]
> Also you removed UnitExplodeSmall from husks which might be undesirable due to it was being used as a visual effect mostly.

That wasn't really intentional, good catch.

Updated.

**Comment by abcdefg30** [comment]
```
Tiberian Dawn Mod at Version {DEV_VERSION}
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.34209
Exception of type `System.Exception`: ActorInfo("v07") failed to initialize because of the following:
Missing:
OpenRA.Mods.Common.Traits.IUpgradableInfo 
Unresolved:
OpenRA.Mods.Common.Traits.UpgradeManagerInfo: { OpenRA.Mods.Common.Traits.IUpgradableInfo }

   at OpenRA.ActorInfo.TraitsInConstructOrder() in \OpenRA\OpenRA.Game\GameRules\ActorInfo.cs:Line 140.
   at OpenRA.Actor..ctor(World world, String name, TypeDictionary initDict) in \OpenRA\OpenRA.Game\Actor.cs:Line 94.
   at OpenRA.World.CreateActor(Boolean addToWorld, String name, TypeDictionary initDict) in \OpenRA\OpenRA.Game\World.cs:Line 241.
   at OpenRA.World.CreateActor(String name, TypeDictionary initDict) in \OpenRA\OpenRA.Game\World.cs:Line 236.
   at OpenRA.Mods.Common.Traits.SpawnMapActors.WorldLoaded(World world, WorldRenderer wr)
   at OpenRA.World.LoadComplete(WorldRenderer wr) in \OpenRA\OpenRA.Game\World.cs:Line 222.
   at OpenRA.Game.StartGame(String mapUID, WorldType type) in \OpenRA\OpenRA.Game\Game.cs:Line 157.
   at OpenRA.Game.LoadShellMap() in \OpenRA\OpenRA.Game\Game.cs:Line 379.
   at OpenRA.Mods.Common.LoadScreens.BlankLoadScreen.StartGame(Arguments args)
   at OpenRA.Game.InitializeMod(String mod, Arguments args) in \OpenRA\OpenRA.Game\Game.cs:Line 366.
   at OpenRA.Game.Initialize(Arguments args) in \OpenRA\OpenRA.Game\Game.cs:Line 255.
   at OpenRA.Program.Run(String[] args) in \OpenRA\OpenRA.Game\Support\Program.cs:Line 109.
   at OpenRA.Program.Main(String[] args) in \OpenRA\OpenRA.Game\Support\Program.cs:Line 39.

**Comment by reaperrr** [comment]
Updated.

**Comment by penev92** [comment]
Looks OK to me :+1:

**Pull request merged**